atlasdb-cassandra/src/main/java/com/palantir/atlasdb/keyvalue/cassandra/CassandraRequestExceptionHandler.java

```diff
@@ -117,13 +117,13 @@ RequestExceptionHandlerStrategy getStrategy() {
     private <K extends Exception> void logNumberOfAttempts(Exception ex, int numberOfAttempts) throws K {
         // Only log the actual exception the first time
         if (numberOfAttempts > 1) {
-            log.info("Error occurred talking to cassandra. Attempt {} of {}. Exception message was: {} : {}",
+            log.debug("Error occurred talking to cassandra. Attempt {} of {}. Exception message was: {} : {}",
                     SafeArg.of("numTries", numberOfAttempts),
                     SafeArg.of("maxTotalTries", maxTriesTotal),
                     SafeArg.of("exceptionClass", ex.getClass().getTypeName()),
                     UnsafeArg.of("exceptionMessage", ex.getMessage()));
         } else {
-            log.info("Error occurred talking to cassandra. Attempt {} of {}.",
+            log.debug("Error occurred talking to cassandra. Attempt {} of {}.",
```

Comment:
:-1: this message has been useful for detecting ongoing issues, e.g. when sweep is misbehaving.

Reply:
retracting this one - we'll log when we error at the last attempt.

Reply:
Do we log more visibly on the last attempt if it fails?

Reply:
Yes, it's a `WARN` if the last attempt fails (see `handleExceptionFromRequest` and `logAndThrowException`.